### ISSUE TITLE
Error: Device logic version too old with version 18

### ISSUE THREAD

**Issue by nuges01** [duguyue100/pyaer#8]
Hi @duguyue100,

I'm trying to run davis346_test.py on a fairly new DAVIS346. However, I get the following error:
```
ERROR: DAVIS ID-1: Device logic version too old. You have version 18; but at least version 9912 is required. Please updated by following the Flashy upgrade documentation at 'https://inivation.com/support/software/reflashing/'.
2019-01-24 22:28:55 (TZ-0500): CRITICAL: DAVIS ID-1: Failed to open device, see above log message for more information (errno=-6).
Traceback (most recent call last):
  File "davis346_test.py", line 14, in <module>
    device = DAVIS(noise_filter=True)
  File "/home/dsmi/anaconda3/envs/neuromorphic/lib/python3.6/site-packages/pyaer/davis.py", line 52, in __init__
    dev_address_restrict, serial_number)
  File "/home/dsmi/anaconda3/envs/neuromorphic/lib/python3.6/site-packages/pyaer/davis.py", line 156, in open
    serial_number)
  File "/home/dsmi/anaconda3/envs/neuromorphic/lib/python3.6/site-packages/pyaer/device.py", line 86, in open
    raise ValueError("The device is failed to open.")
ValueError: The device is failed to open.
```
The error message is a little confusing since 18 is the latest version, but I assume it just means the logic version is incompatible, perhaps with libcaer? I believe install-libcaer.sh installs the latest version of libcaer, but you also pointed out in the other open issue that version 3 will break the API. Do you think that's the cause of this issue?

Thanks!

**Comment by nuges01** [comment]
Actually, looking again, it appears `install-libcaer.sh` gets an older (tagged) version of libcaer, and not the latest one. I wonder if that's what causing the incompatibility? Do you have any pointers on how to get past this? Would you recommend flashing with an older version of the logic and firmware, and if you do, do you know where I can find those files? 

Thanks!

**Comment by nuges01** [comment]
Thanks for pointers. I tried downloading the latest version of libcaer and building it, but I think there's still some compatibility issues, particularly an `undefined symbol` error. It's also possible that the structure as changed because `pyaer._libcaer_wrap` can't be found by python. Below is the full error:

```
$ python davis346_test.py 
Traceback (most recent call last):
  File "/home/dsmi/anaconda3/envs/neuromorphic/lib/python3.6/site-packages/pyaer-0.1.0-py3.6-linux-x86_64.egg/pyaer/libcaer_wrap.py", line 15, in swig_import_helper
    return importlib.import_module(mname)
  File "/home/dsmi/anaconda3/envs/neuromorphic/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 978, in _gcd_import
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 948, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'pyaer._libcaer_wrap'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/dsmi/anaconda3/envs/neuromorphic/lib/python3.6/site-packages/pyaer-0.1.0-py3.6-linux-x86_64.egg/pyaer/__init__.py", line 22, in <module>
    from pyaer import libcaer_wrap as libcaer
  File "/home/dsmi/anaconda3/envs/neuromorphic/lib/python3.6/site-packages/pyaer-0.1.0-py3.6-linux-x86_64.egg/pyaer/libcaer_wrap.py", line 18, in <module>
    _libcaer_wrap = swig_import_helper()
  File "/home/dsmi/anaconda3/envs/neuromorphic/lib/python3.6/site-packages/pyaer-0.1.0-py3.6-linux-x86_64.egg/pyaer/libcaer_wrap.py", line 17, in swig_import_helper
    return importlib.import_module('_libcaer_wrap')
  File "/home/dsmi/anaconda3/envs/neuromorphic/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: /usr/lib/x86_64-linux-gnu/libcaer.so.3: undefined symbol: _ZN2cv8fastFreeEPv

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "davis346_test.py", line 11, in <module>
    from pyaer import libcaer
  File "/home/dsmi/anaconda3/envs/neuromorphic/lib/python3.6/site-packages/pyaer-0.1.0-py3.6-linux-x86_64.egg/pyaer/__init__.py", line 24, in <module>
    raise ImportError("libcaer might not be in the LD_LIBRARY_PATH "
ImportError: libcaer might not be in the LD_LIBRARY_PATH or your numpy might not be the required version.Try to load _libcaer_wrap.so from the package directory, this will provide more information.
```

Perhaps I'll just wait for your upgrade.

Thanks!

**Comment by duguyue100** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Upgrade PyAER to 0.1.1](https://trello.com/c/KBbIK8p8/421-upgrade-pyaer-to-011)

**Comment by duguyue100** [comment]
Hi @nuges01, I've pushed the upgrade to 0.1.1.
You should be able to install it after the CI pushed everything in two hours.

Since I released 0.1.0, `libcaer` changed few bias settings, you could find them out in the issue #7 and my newly prepared bias files in the `scripts/configs` folder.

There are several experimental features that shouldn't affect your development.

I will write everything in the release notes later on.

Let me know if this solves your problem.

**Comment by duguyue100** [comment]
I close this issue since it's solved.
